scripts/pre_commit_linter.py

```diff
@@ -140,7 +149,9 @@ def _pre_commit_linter():
     and pass JSCS binary path
     """
     parsed_args = _PARSER.parse_args()
-    autofix = bool(parsed_args.autofix)
+    input_dir = None
+    if len(parsed_args.directory):
+        input_dir = os.path.join(os.getcwd(), parsed_args.directory[0])
```

Comment:
So I have allowed only one folder to be linted at a time to avoid confusion. and treated arg as string if not passed is Default to None.